inductiva/tasks/methods.py

```diff
@@ -49,3 +50,45 @@ def fetch_task_output(task_id: str,
             result_list = api.download_output(api_instance, task_id, output_dir)
 
             return unpack_output(result_list, output_dir, return_type)
+
+
+def get_task(task_id) -> Dict:
+    api_config = api.validate_api_key(inductiva.api_key)
+
+    with ApiClient(api_config) as client:
+        api_instance = TasksApi(client)
+
+        try:
+            # Get User Tasks
+            api_response = api_instance.get_task(
+                path_params={"task_id": task_id})
+
+            return {**api_response.body}
```

Comment:
This is a bit of "advanced" Python, can you leave here a note on what the type is the api_response.body and why you are using the double-asterisk operator?

Typically, it would be used to expand a dictionary while you are inserting the contents into another dict: (see example from https://bas.codes/posts/python-asterisks)

```python
social_media_details = {
    'twitter': 'bascodes'
}

contact_details = {
    'email': 'blog@bascodes.example.com'
}

user_dict = {'username': 'bas', **social_media_details, **contact_details}
# {'email': 'blog@bascodes.example.com', 'twitter': 'bascodes', 'username': 'bas'}
```

But  in your case, if the "body" was already a dict you could return it directly or make a (deep?) copy...
Can you maybe leave a comment clarifying the type of "body" and if your intent is just to create a dict with a copy of its contents?

Reply:
The response body is a `DynamicSchema` (some data type from the generated client), and I am doing this to convert it to a regular dictionary